src/applications/personalization/profile-2/components/connected-apps/ConnectedApps.jsx

```diff
@@ -0,0 +1,143 @@
+      ? apps.filter(app => app.deleted)
+      : [];
+    const activeApps = apps
+      ? apps.filter(app => !app.deleted)
+      : [];
+
+    const allAppsDeleted = deletedApps.length === apps.length
+
+    return (
+      <div className="va-connected-acct">
+        <h2
+          tabIndex="-1"
+          className="vads-u-margin-y--2 medium-screen:vads-u-margin-bottom--4 medium-screen:vads-u-margin-top--3"
+          data-focus-target
+        >
+          Connected apps
+        </h2>
+
+        {apps && !allAppsDeleted && (
+          <p className="va-introtext vads-u-font-size--md">
+            You’ve given these third-party apps or websites access to some of
+            your Veteran data, like health or service records. You can remove
+            their access at any time by disconnecting the app. Disconnected
+            apps can’t receive any new data from VA, but may still have access
+            to information that you’ve previously shared.
+          </p>
+        )}
+
+        {!apps || allAppsDeleted && (
+          <p className="va-introtext vads-u-font-size--md">
+            You don’t currently have any third-party apps or websites
+            connected to your Veteran data, like health or service records.
+            When you do, this is where you can manage them.
+          </p>
+        )}
+
+        {loading && (
+          <LoadingIndicator setFocus message="Loading your connected apps..." />
+        )}
+
+        {deletedApps.map(app => (
+          <AppDeletedAlert
+            id={app.id}
+            appName={app?.attributes?.title}
+            key={app.id}
+            dismissAlert={this.dismissAlert}
+          />
+        ))}
+
+        {activeApps.map((app, idx) => (
+          <ConnectedApp
+            key={app.id}
+            confirmDelete={this.confirmDelete}
+            isLast={idx + 1 === activeApps.length}
+            {...app}
+          />
+        ))}
+
+        <div className="vads-u-display--flex vads-u-flex-direction--column vads-u-background-color--gray-lightest vads-u-padding--2p5 vads-u-margin-top--2">
+          <h3 className="vads-u-margin--0">
+            Have questions about connecting to VA.gov?
+          </h3>
+          <p>
+            Get answers to frequently asked questions about how connected
+            third-party apps work, what types of information they can see, and
+            the benefits of sharing your information.
+          </p>
+
+          <a
+            className="vads-u-color--primary-alt-darkest"
+            href="/sign-in-faq/"
+            onClick={() =>
+              recordEvent({
+                event: 'account-navigation',
+                'account-action': 'view-link',
+                'account-section': 'vets-faqs',
+              })
+            }
+          >
+            Go to Connected Account FAQs
+          </a>
+
+          <a
+            className="vads-u-color--primary-alt-darkest"
+            href="/third-party-applications/"
```

Comment:
@kelsonic Can you confirm that the public websites team is working on this page and that this is the correct url?

Reply:
@sanlouise Public Websites [has decided not to work on it](https://github.com/department-of-veterans-affairs/va.gov-team/issues/9238#issuecomment-634234268) I believe, though there could be discussions behind the scenes that I don't know about. That is the URL we created for the POC, however, that URL is subject to change after IA review 🙂 

Original ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/9238
POC ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/9419
POC PR: https://github.com/department-of-veterans-affairs/vets-website/pull/12773

Reply:
@mshea0606 Should I remove the link to 'Find apps you can use' until it is clear who takes ownership of publishing that page? Per Kelson's comment above it seems public websites is no longer working on it.

Reply:
@sanlouise Thats correct, the Public Websites team has post-poned this work for the time being.

Reply:
Alright, removed the link! Thanks!